Sasquatch/Config/Sasquatch ObjC.xcconfig

```diff
@@ -2,3 +2,4 @@ GCC_PREPROCESSOR_DEFINITIONS = $(inherited) GCC_PREPROCESSOR_MACRO_SASQUATCH_OBJ
 
 // For verifying projects with disabled modules.
 CLANG_ENABLE_MODULES = NO
+OTHER_LDFLAGS = $(OTHER_LDFLAGS) -framework UserNotifications
```

Comment:
Why do we need this one in the app?

Reply:
To avoid the following error that appears during 'SusquatchObjC' build:

   ```
Undefined symbols for architecture x86_64:
      "_OBJC_CLASS_$_UNUserNotificationCenter", referenced from:
         objc-class-ref in AppDelegate.o
         objc-class-ref in AppCenterPush(MSPush.o)
         objc-class-ref in AppCenterPush(MSUserNotificationCenterDelegateForwarder.o)
```

Reply:
Is the SDK properly declaring the dependency to this system framework? I would think the SDK would infer this dependence. Why don't we have to do that on Sasquatch Swift?

Reply:
That makes sense

Reply:
Wait. Do we really need to add `CLANG_ENABLE_MODULE_DEBUGGING` if we set `CLANG_ENABLE_MODULES = NO` ?

Reply:
Yes, you are right. This property should be removed.